Katello/katello
# Fixes #36866 - Remove table hooks from katello and use foreman

#### What are the changes introduced in this pull request?

Now that TableHooks.js code has been ported to Foreman, we need to have Katello consume it from there so code isn't duplicated. This PR does the clean up / walden job 

#### What are the testing steps for this pull request?

* Check out this PR
* Try some pages that use Table hooks. 
* Make sure they look ok
